### DISCUSSION TITLE
Server disappears from directory

### DISCUSSION THREAD

**Discussion by lolhun1988** [jamulussoftware/jamulus#3256] [Technical Support & Help] [answered]
Hi all,

Thanks to this community succeeded to run a server in a cloud. However I noticed it disappears from the directory after a while and cannot understand why. I would like to get some help.

The server is running properly, please see an output below.

```
F   UID     PID    PPID PRI  NI    VSZ   RSS WCHAN  STAT TTY        TIME COMMAND
4   117    1259       1   0 -20 171712 53632 -      S<sl ?          0:00 /usr/bin/jamulus-headless -s -n --directoryaddress anygenre1.jamulus.io:22124 --serverinfo 🍹Jamsuckers🍹;Amsterdam;NL --welcomemessage Welcome Sucker! --numchannels 20
```

The server can reach the anygenre1.jamulus.io on port 22124.

```
ubuntu@jamulus-server:~$ nc -vzu anygenre1.jamulus.io 22124
Connection to anygenre1.jamulus.io (62.113.204.104) 22124 port [udp/*] succeeded!
```

My server can be reached also via that port.

```
parallels@ubuntu-linux-22-04-desktop:~$ nc -vzu 158.101.201.242 22124
Connection to 158.101.201.242 22124 port [udp/*] succeeded!
```

At this point I am really puzzled what the issue is. What is even weirder if I change to anygenre2 then I change the config back to anygenre1 the server is visible for a while. Then it disappears again after 5-10 minutes.

Have you got any tips or leads for me? Would be greatly appreciated.

**Comment by lolhun1988** [comment]
I just checked jamulus.live and I can see the server but no longer listed in the directory. So the server seems to be running properly, the problem is something else.

The Jamsuckers server is the one.

![image](https://github.com/jamulussoftware/jamulus/assets/165557771/c240bcc9-3c95-45b7-84bb-efc04b164411)

**Reply by mcfnord** [reply]
You wonder why there's no green box for Jamsuckers above, yet Jamsuckers is listed in the Servers list?

This is probably because there's a connection on Jamsuckers, such as a bot or a person who has been there a long time, but it's not active enough to appear in the top list. A connection with only an invisible username(s) would also cause this.

**Comment by lolhun1988** [comment]
OK meanwhile I used this command and noticed this line when I cannot see the server listed anymore. Can it be connected to what I am experiencing?

`journalctl -u jamulus-headless`

`Apr 01 11:25:58 jamulus-server jamulus[3085]: 2024-04-01 11:25:58,, server idling -------------------------------------`

**Reply by dcorson-ticino-com** [reply, marked as answer]
The problem is not your server or the directory.
I have been looking for the last 2 hours and it is always listed and I have connected OK once.
Maybe there is a network problem between you and the directory.

**Reply by lolhun1988** [reply]
Yes, I just noticed I remote into someone else's computer and I could see the server. This is really weird.

**Comment by lolhun1988** [comment]
Even weirder, someone else can see my server but I cannot see it listed in the directory. Uh things are really weird. Sorry for sharing too many updates, I am trying to understand what is going on.

**Reply by dcorson-ticino-com** [reply]
Try using another directories.
The servers are not all located at the same place, although some are located together, I don't remember exactly.
There are apparently network problems between you and the directory server.
